setup.py

```diff
@@ -21,7 +21,7 @@
 DEPENDENCIES = (
     "cachetools>=2.0.0,<5.0",
     "pyasn1-modules>=0.2.1",
-    "rsa>=3.1.4,<5.0",
+    "rsa>=3.1.4,<4.1.0",
```

Comment:
```suggestion
    # TODO: rsa v4.10 doesn't support Python 2. Remove pinning after dropping support for Python 2. 
    "rsa>=3.1.4,<4.1.0",
```

Reply:
Consider the following instead:

# rsa 4.1, 4.1.1, 4.2 are broken on Py2: https://github.com/sybrenstuvel/python-rsa/issues/152
'rsa>=3.1.4,!=4.1,!=4.1.1,!=4.2,<5; python_version < 3',
'rsa>=3.1.4,<5; python_version >= 3',

Reply:
Sure. modified your comment slightly and accepted.

Reply:
Reopening just to make sure you saw my comment before this was resolved.